admin/src/seeds/index.ts

```diff
@@ -1,19 +1,23 @@
+import type { Banner, Event, Space } from '@phits-tech/common/dao-firestore'
 import { BANNERS, EVENTS, SPACES } from '@phits-tech/common/dao-firestore'
 
+import { productionWarning } from '@/_services/modes'
 import { context } from '~/context'
 import { MODE } from '~/modes'
 import migrate from '../migrations/migrate'
 
-import { banners } from './data/banners'
-import { events } from './data/events'
-import { spaces } from './data/spaces'
-
 const db = context.db
 
 const main = async (): Promise<void> => {
-  if (MODE !== 'emu') return console.warn('This script is exclusively for local testing')
+  if (!process.argv.includes('--yes')) {
```

Comment:
I don't like CLI overrides for this warning. You shouldn't be calling this script from CI/CD, so this only creates the risk of pressing up & accidentally re-running on prodUCTION.

Reply:
Done